zapisy/apps/grade/poll/models.py

```diff
@@ -333,6 +333,50 @@ def get_schema_with_default_answers(self):
 
         return {'version': 1, 'schema': []}
 
+    def get_schema_with_random_answers(self):
+        """Fetches the Submission's schema and populates it with randomly generated answers.
+
+        Dedicated for system testing purpose.
+
+        :returns: a schema with additional `answer` keys.
+        """
+        def _random_answer_based_on_question_type(entry):
+            import random
+            import string
+
+            if entry["type"] == "textarea":
+                length = random.randint(1, 400)
+                letters = string.ascii_lowercase
```

Comment:
Tak żeby małym kosztem generowane dane wyglądały choć odrobinę bardziej realistycznie…
```suggestion
                letters = string.ascii_lowercase + " "
```